spec/broadside/ecs/ecs_manager_spec.rb

```diff
@@ -9,28 +9,38 @@
   let(:cluster) { 'cluster' }
   let(:name) { 'job' }
 
-  it 'create_service' do
-    expect { described_class.create_service(cluster, service_name) }.to_not raise_error
+  describe '#create_service' do
```

Comment:
yeah i mean, i don't really care about this part that much/basically fine with this change